core/embed/trezorhal/haptic.h

```diff
@@ -0,0 +1,20 @@
+
+#ifndef __HAPTIC_H__
```

Comment:
This kind of symbol naming is quite inconsistent in our codebase now. In general, we should not introduce identifiers starting with an underscore especially followed by a capital letter, as such symbols are reserved.  In this case, I personally would use TREZORHAL_HAPTIC_H. Maybe we could start using #pragma once.

See https://wiki.sei.cmu.edu/confluence/display/c/DCL37-C.+Do+not+declare+or+define+a+reserved+identifier

Reply:
renamed, see https://github.com/trezor/trezor-firmware/pull/3422/commits/81406b9167c7e244310fd53c1f0cda9b84ac8e20

totally agree with inconsistent usage, out of scope here but i created an issue at least: https://github.com/trezor/trezor-firmware/issues/3425